osu.Game.Rulesets.Catch/Edit/Blueprints/BananaShowerPlacementBlueprint.cs

```diff
@@ -23,6 +23,8 @@ protected override void Update()
             base.Update();
 
             outline.UpdateFrom(HitObjectContainer, HitObject);
+
+            BeginPlacement();
```

Comment:
Why is this one being called every frame?

Reply:
I though I put that in LoadComplete... I'll fix that.